src/rpc/rawtransaction.cpp

```diff
@@ -3078,6 +3078,108 @@ static RPCHelpMan calculateasset()
     };
 }
 
+static RPCHelpMan updatepsbtpegin()
+{
+    return RPCHelpMan{"updatepsbtpegin",
+            "\nFill in Peg-in input data for a particular input in a PSBT. Data is filled if provided.\n"
+            "The Peg-in witness will be constructed and finalized if everything is available.\n",
```

Comment:
This comment is a bit unclear ... it seems to suggest that this RPC will construct the pegin witness and include it in the PSET somewhere, but what you mean is that **during signing** (or finalization? finalization seems to make more sense but in a later commit you implement this in `SignPSBT`) a pegin witness will be constructed only if all fields are available.

Reply:
Removed this line.